Use AlignedSegment.to_string

mvdbeek opened the pull request:
https://pysam.readthedocs.io/en/v0.21.0/api.html?highlight=AlignedSegment#pysam.AlignedSegment.tostring

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

mvdbeek committed:
Use AlignedSegment.to_string

dannon commented on lib/galaxy/datatypes/binary.py:
```diff
@@ -664,7 +664,7 @@ def get_chunk(self, trans, dataset: HasFileName, offset: int = 0, ck_size: Optio
-                        bamline = alignment.tostring(bamfile)
+                        bamline = alignment.to_string(bamfile)
                         # With multiple tags, Galaxy would display each as a separate column
                         # because the 'tostring()' function uses tabs also between tags.
```
```suggestion
                        # because the 'to_string()' function uses tabs also between tags.
```

mvdbeek committed:
Update outdated comment

Co-authored-by: Dannon <dannon.baker@gmail.com>

martenson committed:
fix the call arguments

https://pysam.readthedocs.io/en/v0.21.0/api.html?highlight=AlignedSegment#pysam.AlignedSegment.tostring
